SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -683,3 +669,54 @@ void UGlobalStateManager::SetDeploymentMapURL(const FString& MapURL)
 	UE_LOG(LogGlobalStateManager, Log, TEXT("Setting DeploymentMapURL: %s"), *MapURL);
 	DeploymentMapURL = MapURL;
 }
+
+bool UGlobalStateManager::ProcessOpListForServersCanBeginPlay(const TArray<Worker_OpList*>& OpLists)
+{
+	const Worker_AddComponentOp* AddOp = nullptr;
+	const Worker_AuthorityChangeOp* AuthOp = nullptr;
+
+	for (Worker_OpList* OpList : OpLists)
+	{
+		for (size_t i = 0; i < OpList->op_count; ++i)
+		{
+			const Worker_Op* Op = &OpList->ops[i];
+
+			if (Op->op_type == WORKER_OP_TYPE_ADD_COMPONENT &&
+				Op->add_component.data.component_id == SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID)
+			{
+				AddOp = &Op->add_component;
+			}
+			else if (Op->op_type == WORKER_OP_TYPE_AUTHORITY_CHANGE &&
+				Op->authority_change.component_id == SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID)
+			{
+				AuthOp = &Op->authority_change;
+			}
+		}
+	}
+
+	// verify assumption that we should never receive only the auth op here
+	check(AddOp || AuthOp == nullptr);
+
+	if (AddOp == nullptr)
+	{
+		return false;
+	}
+
```

Comment:
Since CanBeginPlay is no longer used to gate BeginPlay, we should consider renaming it.  The actual usage now is just to track whether or not we have called BeginPlay with Authority.  AuthBeginPlayCalled?